tests/functional/object/mcg/ui/test_s3_vector_ui.py

```
@@ -0,0 +1,477 @@
+                bucket_name=bucket_name,
+                storageclass=storageclass,
+                bucketclass_name=bucketclass_name,
+            )
+            # Wait for the OBC controller to populate the NooBaa bucket name
+            # before reading it; reading immediately after form submission can
+            # race on slower clusters.
+            noobaa_bucket_name = None
+            for noobaa_bucket_name in TimeoutSampler(
+                timeout=60,
+                sleep=3,
+                func=lambda: OBC(bucket_name).bucket_name,
```

Ruff (0.15.15)

[warning] Function definition does not bind loop variable `bucket_name`

(B023)